
Don't open extension modules with RTLD_LAZY: better to fail obviously at
load-time rather than obscurely at run-time if they have undefined 
symbols.

--- Zend/zend.h	2004-03-16 18:36:17.000000000 +0100
+++ Zend/zend.h.oden	2004-05-04 18:01:30.353052855 +0200
@@ -92,7 +92,7 @@
 # if defined(RTLD_GROUP) && defined(RTLD_WORLD) && defined(RTLD_PARENT)
 #  define DL_LOAD(libname)			dlopen(libname, RTLD_LAZY | RTLD_GLOBAL | RTLD_GROUP | RTLD_WORLD | RTLD_PARENT)
 # else
-#  define DL_LOAD(libname)			dlopen(libname, RTLD_LAZY | RTLD_GLOBAL)
+#  define DL_LOAD(libname)			dlopen(libname, RTLD_NOW | RTLD_GLOBAL)
 # endif
 # define DL_UNLOAD					dlclose
 # if DLSYM_NEEDS_UNDERSCORE
